### PR TITLE
[6.0] Convert Checkbox to latest mergeStyles pattern

### PR DESCRIPTION
**Pull request by jordandrako**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

##### Convert Checkbox to mergeStyles

Checkbox was using the old method of mergeStyles (classNames file) and needed some cleanup.
- Ensured Checkbox matches the XD design doc (both individual Checkboxes and within Dropdowns)
  - Dropdown was using ref on Checkbox and styling incorrectly
- Simplified and combined all styling logic to the styles.ts file.
- Fixed customStyles for customization. (Remove for other customization methods?)

#### Focus areas to test

Ensure the checkbox works as intended.

### CODE REVIEW

**Comment by jordandrako** [comment]
Oi, so since this branch was created from master, this is messy.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,96 +4,102 @@
...
-# *.js    @js-owner
+# *.js@js-owner
 
 # You can also use email addresses if you prefer. They'll be used to look up
 # users just like we do for commit author emails.
-# docs/*  docs@example.com
+# docs/  docs@example.com
 
 #### Apps
-# component-demo/*
```
Comment (dzearing) [inline .github/CODEOWNERS]: Can you make changes to this in master, and maybe separately?

File: .vscode/launch.json [hunk 2]
```diff
@@ -22,14 +22,14 @@
       "sourceMaps": true
     },
     {
-      "name": "Debug test in experiments",
+      "name": "Build fabric",
       "type": "node",
       "request": "launch",
       "program": "${workspaceRoot}/scripts/build.js",
-      "cwd": "${workspaceRoot}/packages/experiments",
```
Comment (dzearing) [inline .vscode/launch.json]: Can you revert, or change master?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,14 +1,58 @@
 # Contributing to Office UI Fabric
 
-## Creating New Components
+## Components
+Our components are a collection of small, independent, general-purpose UI components intended to be reused throughout an existing application.
 
-### Creating a component via the command line
+### Contributing new components
+We have two main buckets for our new and existing components, Experimental and Fundamental. These categories are created to notify Fabric users which components are expected to be fully supported and stable and which ones are likely to change drastically over a short period of time.
```
Comment (dzearing) [inline CONTRIBUTING.md]: again, master if possible.

File: apps/fabric-website/CHANGELOG.json [hunk 4]
```diff
@@ -1,6 +1,146 @@
...
+            "author": "lynamemi <lynam.emily@gmail.com>",
+            "commit": "a9d0f70495f315998a5dad78699ffd2349436420",
+            "comment": "Adding left nav scroll functionality, including fixing the left nav with the header. Other layout refactoring to accommodate."
+          }
+        ],
+        "dependency": [
+          {
+            "comment": "Updating dependency \"office-ui-fabric-react\" from `>=5.40.1 <6.0.0` to `>=5.41.0 <6.0.0`"
+          }
+        ]
```
Comment (dzearing) [inline apps/fabric-website/CHANGELOG.json]: this shouldn't be in this PR.

File: apps/fabric-website/package.json [hunk 5]
```diff
@@ -18,34 +18,28 @@
...
+    "@types/es6-promise": "0.0.32",
     "@types/node": "8.0.26",
-    "@types/prop-types": "15.5.1",
+    "@types/prop-types": "15.5.2",
     "@types/resemblejs": "~1.3.28",
-    "@types/sinon": "2.2.2",
-    "@types/react": "15.0.38",
-    "@types/react-dom": "15.5.1",
+    "@types/react": "16.0.25",
+    "@types/react-dom": "16.0.3",
```
Comment (dzearing) [inline apps/fabric-website/package.json]: I am thinking maybe we should close this PR, and reopen one that has the change you intended :)

**Comment by dzearing** [comment]
I'm going to close this. It doesn't look quite right... Can you adjust the PR to either target master and have a few targeted changes, or 6.0 and have a few targeted changes?

I have a merge of master into 6.0 which should help

**Pull request closed without merging**